mountpoint-s3/src/fs.rs

```diff
@@ -1016,6 +1024,7 @@ where
                     return Ok(reply);
                 }
             }
```

Comment:
We don't need this `last_response` case any more, right?

Reply:
Decided to leave it since removing it might break existing application that were relying on this behavior.